src/boot/mexpr.ml

```diff
@@ -147,7 +147,7 @@ let fail_constapp f v fi = raise_error fi ("Incorrect application. function: "
    The reason for this is that if-expressions return expressions
    and not values. *)
 let delta eval env fi c v  =
-    let index_out_of_bounds_in_seq_msg = "Out of bound access in sequence." in
+    let out_of_bounds_in_seq_msg = "Out of bounds." in
```

Comment:
Sorry, I was unclear! My comment was that it should be "Out of bounds" and not "out of bound". I think the message should be "Out of bounds access in sequence". Just out of bounds is a bit unspecific.

Reply:
Fixed